migrations/20200610113619-update_list_assessment_programs_query.js

```diff
@@ -0,0 +1,31 @@
+const Uuid = require('cassandra-driver').types.Uuid;
+
+module.exports = {
+  async up(db) {
+    global.migrationMsg = "Update list observation solutions query";
+    
+    
+    if (!cassandra) {
+      throw new Error("Cassandra connection not available.");
+    }
+
+    const query = 'SELECT id FROM reports.druidqueries WHERE qid = ? ALLOW FILTERING';
+    const result = await cassandra.execute(query, [ 'list_assessment_programs_query' ], { prepare: true });
+    const row = result.rows;
+   
+   
+    let queries = 
+      [{
+        query: 'UPDATE reports.druidqueries SET query=? WHERE id=?',
+        params: ['{"queryType":"groupBy","dataSource":"sl_assessment_dev","granularity":"all","dimensions":["programName","programId","programDescription","programExternalId","solutionName","solutionId","solutionDescription","solutionExternalId"],"filter":{"type":"and","fields":[{"type":"selector","dimension":"","value":""},{"type":"or","fields":[{"type":"and","fields":[{"type":"selector","dimension":"userId","value":""},{"type":"selector","dimension":"isAPrivateProgram","value":true}]},{"type":"selector","dimension":"isAPrivateProgram","value":false}]}]},"aggregations":[],"postAggregations":[],"intervals":["1901-01-01T00:00:00+00:00/2101-01-01T00:00:00+00:00"]}',row[0].id]
```

Comment:
@deepakr123 - let's use the name as sl_assessment

Reply:
resolved